utils/check.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Module that check errors
+module Check
+  def empty?(*_variables)
+    _variables.each { |variable| raise EmptyError, "#{variable} is empty!" if variable.nil? }
+  end
+
+  def positive?(*_variables)
+    _variables.each { |variable| raise InputError, "#{variable} is negative!" if variable.negative? }
+  end
+
+  def string?(*_variables)
+    _variables.each { |variable| raise TypeError, "#{variable} isn't string!" unless variable.is_a? String }
+  end
+
+  def integer?(*_variables)
```

Comment:
Lint/UnderscorePrefixedVariableName: Do not use prefix `_` for a variable that is used.